collections/without.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+
+/** Used as the size to enable large array optimizations. */
+const LARGE_ARRAY_SIZE = 200;
+
+/**
+ * Returns an array excluding all given values.
+ *
+ * Example:
+ *
+ * ```ts
+ * import { without } from "./without.ts";
+ *
+ * const withoutList = without([2, 1, 2, 3], 1, 2);
+ *
+ * console.assert(withoutList === [ 3 ])
+ * ```
+ */
+export function without<T>(
+  arrays: Array<T>,
+  ...values: Array<T>
+): T[] {
+  if (values.length >= LARGE_ARRAY_SIZE) {
+    const set = new Set<T>();
+    for (const element of arrays) {
+      set.add(element);
+    }
+    arrays = Array.from(set);
```

Comment:
This makes the array items distinct. Is this really what you wanted to do?

Reply:
This looks wrong to me. I would also suggest renaming the argument to `array`, as it is actually just one array and maybe create a new variable, as shadowing arguments can makes things harder to follow.

Reply:
fixed